sql/moz-fx-data-shared-prod/default_browser_agent_derived/default_browser_agg_v1/query.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
+  DATE(submission_timestamp) AS submission_date,
+  mozfun.norm.truncate_version(build_version, 'major') AS build_version_major,
+  mozfun.norm.get_windows_info(`os_version`).name AS windows_name,
```

Comment:
Not sure how important it is, but there is this one chart that contains the full version number:
https://sql.telemetry.mozilla.org/queries/70146/source?p_date=2022-11-24#176653